index.ftd

```diff
@@ -1582,36 +1478,26 @@ menu-hover-bg: $menu-hover-bg
 
 -- ftd.column show-children-mobile:
 fpm.toc-item list children:
-ftd.color menu-bg:
-ftd.color menu-hover-bg:
-ftd.color menu-link:
-ftd.color menu-link-active:
-ftd.color menu-border:
 if: $children is not empty
 padding: 5
 padding-left: 15
 
 --- l-mobile:
 obj: $obj
 $loop$: $children as $obj
-menu-link: $menu-link
-menu-link-active: $menu-link-active
+
 
 
 
 -- ftd.column l:
 fpm.toc-item obj:
-ftd.color menu-hover-bg:
-ftd.color menu-link:
-ftd.color menu-link-active:
 width: fill
 
 --- ftd.text: $obj.title
 if: $obj.url is not null
 link: $obj.url
-color: $menu-link
+color: $fpm.color.main.background.step-2
```

Comment:
@priyanka9634 static color shouldn't be mentioned like this in a package component - remove whole component OR keep it as is